e2e-nodejs/group-pkp-cosmos/test-pkp-cosmos-stargate-client.mjs

```diff
@@ -12,7 +12,7 @@ import {
 } from '@cosmjs/stargate';
 
 const TX_HASH =
-  '0D37992882EAF76E0C914B55CFACFDDDA17C9D8B698A6FDB171038AE05B91AB5';
+  '6E7CA7AC2A30FAA852DB67E85284E7EE670E581CBFD06DE104AF00C2A1796A1B';
```

Comment:
why do we need this hard coded tx hash?

Reply:
just checking if it could use the client to look up a tx, it can be any, but the previous stopped working probably cus they wiped the testnet